src/rf/apps/workers/process.py

```diff
@@ -367,16 +383,28 @@ def emr_event(self, record, started_status, finished_status):
             return False
 
         if status == FAILED:
-            error = record.get('error', '')
-            return status_updates.update_layer_status(layer_id,
-                                                      enums.STATUS_FAILED,
-                                                      error)
+            default_error = emr_status + ' job failed.'
+            error = record.get('error', default_error)
+            if error.strip() == '':
+                error = default_error
```

Comment:
But in this case, chunk/mosaic errors are stored in `status_failed_error` instead of their respective error fields. Do we ever use `status_mosaic_error` or `status_chunk_error` then? I'll test this.